contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/azkaban/VoldemortBuildAndPushJob.java

```diff
@@ -830,7 +835,8 @@ public void runPushStore(Props props, String url, String dataDir) throws Excepti
                 maxNodeFailures,
                 failedFetchStrategyList,
                 url,
-                buildPrimaryReplicasOnly).run();
+                buildPrimaryReplicasOnly,
+                props).run();
     }
```

Comment:
Instead of triggering distcp in 'VoldemortSwapJob', could we trigger it in 'VoldemortBuildAndPushJob'? 
It is more clear to me to have three steps in the main class:
build -> copy -> fetch/swap

What do you think?